packages/metadata/src/inspector.ts

```diff
@@ -23,6 +23,14 @@ export interface DesignTimeMethodMetadata {
   returnType: Function;
 }
 
+/**
+ * Options for inspection
+ */
+export interface InspectionOptions {
+  // Only inspect own metadata of a given target
```

Comment:
I think it would be better to use TSDoc comment here.

```ts
  /**
   * Inspect own metadata of a given target only,
   * ignore any metadata inherited from parent classes.
   * By default, inspector returns both own and inherited metadata.
   */
   ownMetadataOnly?: boolean;